llvm/lib/MC/MCCASObjectV1.cpp

```diff
@@ -482,9 +528,25 @@ DebugInfoSectionRef::create(MCCASBuilder &MB,
   return get(B->build());
 }
 
+Expected<DebugAbbrevSectionRef>
+DebugAbbrevSectionRef::create(MCCASBuilder &MB,
+                              ArrayRef<cas::ObjectRef> Fragments) {
+  Expected<Builder> B = Builder::startNode(MB.Schema, KindString);
+  if (!B)
+    return B.takeError();
+
+  if (auto E = encodeReferences(Fragments, B->Data, B->Refs))
+    return std::move(E);
+
+  writeRelocationsAndAddends(MB.getSectionRelocs(), MB.getSectionAddends(),
+                             B->Data);
+  return get(B->build());
+}
+
 /// Class that represents a fully verified DebugInfoSectionRef object loaded
 /// from the CAS.
 struct LoadedDebugInfoSection {
+  StringRef FixupsData;
   SmallVector<size_t> AbbrevOffsets;
```

Comment:
, 0?

Reply:
This was added by @felipepiovezan. Any good reason for leaving out the 0?

Reply:
The reason is that I lacked a well informed choice for this!

> In the absence of a well-motivated choice for the number of inlined elements N, it is recommended to use SmallVector<T> (that is, omitting the N). This will choose a default number of inlined elements reasonable for allocation on the stack (for example, trying to keep sizeof(SmallVector<T>) around 64 bytes).

https://llvm.org/docs/ProgrammersManual.html#llvm-adt-smallvector-h


A reason to use 0 would be if, for example, we have reason to believe this is almost always empty or if we will rely on move semantics often.

Reply:
This is not technically part of this patch, so maybe an NFC patch?